modules/API/API.cpp

```diff
@@ -474,10 +475,11 @@ void API::init() {
         });
 
         std::string cmd_set_limit = cmd_base + "set_limit_amps";
-        this->mqtt.subscribe(cmd_set_limit, [&evse](const std::string& data) {
+        this->mqtt.subscribe(cmd_set_limit, [&evse, &r_evse_manager_energy_sink = this->r_evse_manager_energy_sink,
```

Comment:
Note: `&evse` isn't used here at all.  Furthermore, the lambda only cares about `r_evse_manager_energy_sink.at(evse_id - 1)`, why not *calculating* this reference beforehand and probably do some sanity checking (if it exists and matches for example) - it is used in both lambdas.